### PR TITLE
ArcSight Logger:

### PR DESCRIPTION
**Pull request by dantavori**
## Status
In Hold (no instance, waiting for customer approval for bug fix)

## Related Issues
fixes: [https://github.com/demisto/etc/issues/18878](https://github.com/demisto/etc/issues/18878)

## Description
Fixed an issue where date fields in search results were in epoch format, and not human readable.

## Does it break backward compatibility?
   - No

## Technical writer review
- [ ] [CHANGELOG](https://github.com/demisto/content/blob/ArcSightLogger/Integrations/integration-ArcSightLogger_CHANGELOG.md)

### CODE REVIEW

**Comment by kirbles19** [comment]
@dantavori Done, no changes.

File: Integrations/integration-ArcSightLogger.yml [hunk 1]
```diff
@@ -540,15 +540,22 @@ script:
...
                 if(field.name){
                     keys.push(field.name.replace(/\s/g, '').replace(/^\_/,''));
+                    isDateField.push(field.type == 'date');
                 }
             });
             entries.forEach(function(entry){
                 if (entry.length == keys.length){
                     var newEntry = {};
                     for (var i = 0; i<  entry.length; i++){
+                        if (isDateField[i]){
```
Comment (glicht) [inline Integrations/integration-ArcSightLogger.yml]: Let's make sure we have an int
```suggestion
                        if (isDateField[i] && !isNaN(entry[i])) {
```

**Comment by ronykoz** [comment]
@glicht can we merge this? seems pretty straight forward for me, although we didn't test this because of instance issues.

**Comment by glicht** [comment]
you can merge

**Pull request merged**